rest-server/src/controllers/job.js

```diff
@@ -24,7 +24,15 @@ const logger = require('../config/logger');
  */
 const load = (req, res, next, jobName) => {
   new Job(jobName, (job) => {
-    if (job.jobStatus.state === 'JOB_NOT_FOUND' && req.method !== 'PUT') {
+    if (Object.keys(job).length === 1) {
```

Comment:
why do we need to check launcher here?

Reply:
If rest-server cannot connect to launcher, `job.jobStatus` will be `undefined`, thus rest-server will crash here.